portal/development_gulpfile.js

```diff
@@ -157,14 +157,12 @@ gulp.task("gilLess", () => {
         .pipe(postCSS())
         .pipe(rename(GIL + ".css"))
         .pipe(sourcemaps.write("../../../"+mapPath)) //path relative to the source file
-        .pipe(gulp.dest("gil/" + cssPath));
-
-    setTimeout(function() {
-        replaceStd(GIL + ".css.map");
-    }, 500);
+        .pipe(gulp.dest("gil/" + cssPath))
```

Comment:
If you need to access something outside `rootPath` (ie `static`) you should probably change `rootPath` to a higher directory. 

Can you rewrite `rootPath` or `cssPath` as necessary so everything can be relative (subdirectory) to `rootPath`?

Decent example of using using `process.cwd()` to set something similar to `rootPath` [here](https://stackoverflow.com/questions/30594308/how-can-i-get-the-path-from-where-gulp-was-run/30595657#30595657)

Reply:
@ivan-c ivan, it is a bit complicated - dealing with multiple entry points here:  ./static/, ./gil/static, ./eproms/static,  to each of the destination that the gulp task generates the css file. I rewrote to have a central rootPath, "./" to but still need to diverge to each (EPROMsPath and GILPath) for the generated css file.